src/main/java/com/github/wovnio/wovnjava/Lang.java

```diff
@@ -57,22 +57,22 @@
         map.put("fa",     new Lang("fa", "fa", "Persian", "زبان_فارسی"));
         map.put("pl",     new Lang("pl", "pl", "Polish", "Polski"));
         map.put("pt",     new Lang("pt", "pt", "Portuguese", "Português"));
-        map.put("pt-BR",  new Lang("pt-BR", "pt-BR", "Portuguese (Brazil)", "Português (Brasil)"));
-        map.put("pt-PT",  new Lang("pt-PT", "pt-PT", "Portuguese (Portugal)", "Português (Portugal)"));
+        map.put("pt-br",  new Lang("pt-BR", "pt-BR", "Portuguese (Brazil)", "Português (Brasil)"));
+        map.put("pt-pt",  new Lang("pt-PT", "pt-PT", "Portuguese (Portugal)", "Português (Portugal)"));
         map.put("ru",     new Lang("ru", "ru", "Russian", "Русский"));
         map.put("es",     new Lang("es", "es", "Spanish", "Español"));
-        map.put("es-RA",  new Lang("es-RA", "es-RA", "Spanish (Argentina)", "Español (Argentina)"));
-        map.put("es-CL",  new Lang("es-CL", "es-CL", "Spanish (Chile)", "Español (Chile)"));
-        map.put("es-CO",  new Lang("es-CO", "es-CO", "Spanish (Colombia)", "Español (Colombia)"));
-        map.put("es-CR",  new Lang("es-CR", "es-CR", "Spanish (Costa Rica)", "Español (Costa Rica)"));
-        map.put("es-HN",  new Lang("es-HN", "es-HN", "Spanish (Honduras)", "Español (Honduras)"));
+        map.put("es-ra",  new Lang("es-RA", "es-RA", "Spanish (Argentina)", "Español (Argentina)"));
+        map.put("es-cl",  new Lang("es-CL", "es-CL", "Spanish (Chile)", "Español (Chile)"));
+        map.put("es-co",  new Lang("es-CO", "es-CO", "Spanish (Colombia)", "Español (Colombia)"));
+        map.put("es-cr",  new Lang("es-CR", "es-CR", "Spanish (Costa Rica)", "Español (Costa Rica)"));
+        map.put("es-hn",  new Lang("es-HN", "es-HN", "Spanish (Honduras)", "Español (Honduras)"));
         map.put("es-419", new Lang("es-419", "es-419", "Spanish (Latin America)", "Español (Latinoamérica)"));
-        map.put("es-MX",  new Lang("es-MX", "es-MX", "Spanish (Mexico)", "Español (México)"));
-        map.put("es-PE",  new Lang("es-PE", "es-PE", "Spanish (Peru)", "Español (Perú)"));
-        map.put("es-ES",  new Lang("es-ES", "es-ES", "Spanish (Spain)", "Español (España)"));
-        map.put("es-US",  new Lang("es-US", "es-US", "Spanish (United States)", "Español (Estados Unidos)"));
-        map.put("es-UY",  new Lang("es-UY", "es-UY", "Spanish (Uruguay)", "Español (Uruguay)"));
-        map.put("es-VE",  new Lang("es-VE", "es-VE", "Spanish (Venezuela)", "Español (Venezuela)"));
```

Comment:
This map requiring lowercase keys is a bit fragile. If you want, you could refactor it to be more like this:
```
        ArrayList<Lang> langs = new ArrayList<Lang>();
        langs.add(new Lang("ar", "ar", "Arabic", "ﺎﻠﻋﺮﺒﻳﺓ"));
        //...
        HashMap<String, Lang> map = new HashMap<String, Lang>();
        for(Lang lang: langs){
            map.put(lang.code.toLowerCase(), lang);
         } 
        LANG = Collections.unmodifiableMap(map);
```